sources/packages/backend/libs/test-utils/src/factories/application.ts

```diff
@@ -80,7 +84,10 @@ export async function saveFakeApplicationDisbursements(
   },
   options?: {
     applicationStatus?: ApplicationStatus;
+    offeringIntensity?: OfferingIntensity;
     createSecondDisbursement?: boolean;
+    disbursementScheduleStatusA?: DisbursementScheduleStatus;
```

Comment:
Just wondering if we would like to pass a second parameter to the `disbursementScheduleA` how it would look?
```ts
options?: {
  applicationStatus?: ApplicationStatus;
  offeringIntensity?: OfferingIntensity;
  createSecondDisbursement?: boolean;
  disbursementScheduleStatusA?: DisbursementScheduleStatus;
  disbursementScheduleMyOtherProperyA?: string;
  disbursementScheduleStatusB?: DisbursementScheduleStatus;
  disbursementScheduleMyOtherProperyB?: string;
},
```